Tests/Tests/STPSourceInfoViewControllerTest.m

```diff
@@ -0,0 +1,109 @@
+//
+//  STPSourceInfoViewControllerTest.m
+//  Stripe
+//
+//  Created by Ben Guo on 2/13/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "STPSourceInfoViewController.h"
+#import "STPSourceInfoDataSource.h"
+#import "STPTextFieldTableViewCell.h"
+
+@interface STPSourceInfoViewController ()
+@property(nonatomic)STPSourceInfoDataSource *dataSource;
+@end
+
+@interface STPSourceInfoViewControllerTest : XCTestCase
+
+@end
+
+@implementation STPSourceInfoViewControllerTest
+
+- (STPSourceInfoViewController *)sutWithParams:(STPSourceParams *)params {
```

Comment:
ah sorry, i use that jargon out of habit:
https://en.wikipedia.org/wiki/System_under_test